Dockerfile.in

```diff
@@ -30,6 +30,7 @@ COPY ${REMOTE_SOURCE}/main.go .
 COPY ${REMOTE_SOURCE}/apis ./apis
 COPY ${REMOTE_SOURCE}/controllers ./controllers
 COPY ${REMOTE_SOURCE}/internal ./internal
+COPY ${REMOTE_SOURCE}/test ./test
```

Comment:
We don't want test code in the operator build

Reply:
+1 my bad